src/tests/ftest/control/config_generate_output.py

```diff
@@ -352,64 +426,59 @@ def test_num_engines(self):
 
         self.check_errors(errors)
 
-    def test_min_ssds(self):
-        """Test --min-ssds.
+    def test_scm_only(self):
+        """Test --scm-only.
 
-        1. Iterate the NVMe PCI dictionary and find the key that has the
-        shortest list. This would be our min_ssd engine count threshold.
-        2. Call dmg config generate --min-ssds=<1 to min_ssd>. Should pass.
-        3. Call dmg config generate --min-ssds=<min_ssd + 1>. Should fail.
-        4. Call dmg config generate --min-ssds=0. Iterate the engines field and
+        1. Call dmg config generate --scm-only=False. Iterate the engines field and
+        verify that there's a bdev_list field.
+        2. Call dmg config generate --scm-only=True. Iterate the engines field and
         verify that there's no bdev_list field.
 
         :avocado: tags=all,full_regression
         :avocado: tags=hw,large
         :avocado: tags=control,dmg_config_generate
-        :avocado: tags=ConfigGenerateOutput,test_min_ssds
+        :avocado: tags=ConfigGenerateOutput,test_scm_only
         """
         # Get necessary storage and network info.
         self.prepare_expected_data()
 
-        # Iterate the NVMe PCI dictionary and find the key that has the shortest
-        # list. This would be our min_ssd engine count threshold.
-        socket_ids = list(self.nvme_socket_to_addrs.keys())
-        shortest_id = socket_ids[0]
-        shortest = len(self.nvme_socket_to_addrs[shortest_id])
-        for socket_id in socket_ids:
-            if len(self.nvme_socket_to_addrs[socket_id]) < shortest:
-                shortest = len(self.nvme_socket_to_addrs[socket_id])
-                shortest_id = socket_id
-
-        min_ssd = len(self.nvme_socket_to_addrs[shortest_id])
-        self.log.info("Maximum --min-ssds threshold = %d", min_ssd)
-
         dmg = DmgCommand(self.bin)
         dmg.exit_status_exception = False
 
         errors = []
 
-        # Call dmg config generate --min-ssds=<1 to min_ssd>. Should pass.
-        for num_ssd in range(1, min_ssd + 1):
-            result = dmg.config_generate(
-                access_points="wolf-a", min_ssds=num_ssd, net_provider=self.def_provider)
-            if result.exit_status != 0:
-                errors.append("config generate failed with min_ssd = {}!".format(num_ssd))
-
-        # Call dmg config generate --min_ssds=<min_ssd + 1>. Should fail.
+        # Call dmg config generate with --scm-only=False
         result = dmg.config_generate(
-            access_points="wolf-a", min_ssds=min_ssd + 1, net_provider=self.def_provider)
-        if result.exit_status == 0:
-            errors.append("config generate succeeded with min_ssd + 1 = {}!".format(min_ssd + 1))
+            access_points="wolf-a", scm_only=False, net_provider=self.def_provider)
```

Comment:
Not sure to fully understand test test suite. However, is this test limited to the wolf test cluster A as the acces_points parameters is hard coded?

Reply:
OK, thanks for the explanation.